src/components/Button.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import classnames from 'classnames';
+
+const Button = ({ text, variant, type, onClick, withIcon, className }) => {
+	const variantClasses = {
+		primary:
+			'bg-green dark:bg-light-blue text-white hover:bg-green-dark active:scale-105 dark:text-black',
+		secondary:
+			'bg-gray-300 text-gray-600 hover:bg-gray-400 dark:text-white dark:bg-gray-700 active:scale-105',
```

Comment:
In our figma, the font color and line for the secondary buttons is dark green in light mode and light blue in dark mode. Should we align it with that? I'd also consider adding a little space between the line and the text, perhaps `underline-offset-4` or `underline-offset-8`.

Reply:
Good catch @polly89. I agree it will be helpful if the secondary buttons can be updated on this PR to match Figma.